org.openmw.OpenMW.json

```diff
@@ -0,0 +1,360 @@
+{
+  "app-id": "org.openmw.OpenMW",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "branch": "master",
+  "command": "openmw-launcher",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc", "--socket=x11", "--device=dri",
+    "--socket=pulseaudio"
+  ],
+  "build-options" : {
+    "cflags": "-O2 -g",
+    "cxxflags": "-O2 -g"
+  },
+  "cleanup": ["/include", "/lib/pkgconfig",
+    "/share/pkgconfig", "/share/aclocal",
+    "/man", "/share/man", "/share/gtk-doc",
+    "/share/vala",
+    "*.la", "*.a"],
+    "modules": [
+      {
+        "name": "libmp3lame",
```

Comment:
should use lame [from shared-modules](https://github.com/flathub/shared-modules/blob/master/lame/lame-3.99.5.json).